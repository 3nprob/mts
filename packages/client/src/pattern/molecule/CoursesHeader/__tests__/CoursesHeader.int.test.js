import { defaultRender } from './CoursesHeader.int.render'

describe('<CoursesHeader />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
