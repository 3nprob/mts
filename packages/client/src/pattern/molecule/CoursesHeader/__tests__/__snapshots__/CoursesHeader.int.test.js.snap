// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoursesHeader /> Render must match regular() 1`] = `
<div
  className=""
>
  <div
    className="container"
  >
    <div
      className="containerLogo"
    >
      <div
        className="containerLogoImage"
      />
    </div>
    <div
      className="containerSessionLinks"
    >
      <div
        className="containerSessionLinksMenu"
      >
        <div>
          <span
            className="containerSessionLinksMenuItem"
            role="menuitem"
            tabIndex="0"
          >
            <div>
              <div
                className="containerSessionLinksMenuItemText"
              >
                <div>
                  My Courses
                </div>
              </div>
            </div>
          </span>
        </div>
        <div
          className="containerSessionLinksMenuSelect"
        >
          <button
            className="containerSessionLinksMenuSelectButton"
            tabIndex="0"
            type="button"
          >
            <div>
              <svg
                className="containerSessionLinksMenuSelectButtonImage"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
